src/plugins/telemetry/server/config/telemetry_labels.ts

```diff
@@ -34,7 +35,7 @@ export const labelsSchema = schema.object(
      */
     serverless: offeringBasedSchema({ serverless: schema.maybe(schema.string()) }),
   },
-  { defaultValue: {} }
+  { unknowns: 'allow', defaultValue: {} }
```

Comment:
Allowing unknowns is probably redundant, however, there's no harm in doing so.

Reply:
Technically this adds a risk that someone could be running an invalid configuration that is silently not working (e.g. due to a typo). However in this case since it is something we use internally for telemetry, I don't think that's particularly concerning. Plus it would help us avoid future config errors at the orchestration level. I'm fine merging with this but if @afharo feels differently maybe we can discuss more next week and consider removing it as a follow-up.

Reply:
FWIW, we were explicit to minimize security concerns: since anything in these labels extend to the UI, unintentional leaks may occur if admins add non-public info there. This doesn't seem like a setting admins may want to change at all, but the risk is still there.

I'm happy with this change as long as @elastic/kibana-security approves 😇

Reply:
Heya team! If it's not required for the functionality, I think it should be removed. ++ accidental data exposure to the UI could be a concern.

Is there significant benefit to having `unknowns`?

Reply:
Ah, had not noticed we were exposing this to the UI as well. In that case ++ to removing it